projects/plugins/boost/changelog/update-jetpack-boost-shield-site-urls

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Change this to `patch` to stop the changelogger complaining. We will manage our versions as we do releases; I don't want individual PRs messing with versions :)

Reply:
Thanks! Fixed with ec32e0dbb5edbafd4881e58b8104afa01d62fd07.

I was thinking of doing it for the next PR, but didn't think to do it in this one.